spec/services/idv/steps/ipp/ssn_step_spec.rb

```diff
@@ -44,26 +44,5 @@
         expect(session[:idv]['applicant']).to be_blank
       end
     end
-
```

Comment:
Can you add these back so they test `step.extra_view_variables` instead of `step.call`?

Reply:
So changing `.call` in the tests to `.extra_view_variables` applies the test to a later step in the workflow?